tutos/dev/manual/application.wiki

```diff
@@ -866,11 +900,28 @@ We then add the slider to the page body, as follows:
 
 <<code language="ocaml" class="server"|
 let page () =
-  (html
-    (head (title (txt "Graffiti")) [])
-    (body [h1 [txt "Graffiti"];
-           canvas_elt;
-           div [slider]] ))
+  let colorpicker, cp_sig =
+    Ot_color_picker.make ~a:[Html.D.a_class ["colorpicker"]]
+  in
+  let slider, slider_sig = slider_create () in
+  ( Html.D.html
+      (Html.D.head
+         (Html.D.title (Html.D.txt "Graffiti"))
+         [ Html.D.css_link
+             ~uri:
+               (Html.D.make_uri
+                  (Eliom_service.static_dir ())
+                  ["css"; "graffiti.css"])
+             ()
+         ; Html.D.css_link
+             ~uri:
+               (Html.D.make_uri
+                  (Eliom_service.static_dir ())
+                  ["css"; "ot_color_picker.css"])
+             () ])
+      (Html.D.body [canvas_elt; slider; colorpicker])
+  , cp_sig
+  , slider_sig )
```

Comment:
All this is much more complicated than before.
The goal of this tutorial is to show that you can do that in very few lines of code.
Can you simplify?